src/components/comments.jsx

```diff
@@ -14,6 +15,20 @@ const Comments = () => {
         });
     }, [review_id]);
 
+    if (comments.length === 0) {
+        return (
+            <div>
+                <h3>This review has no comments...</h3>
+                <form>
+                    <label>Add a comment: </label>
+                    <br></br>
+                    <input type="text" id="comment_body" />
+                    <input type="submit" />
+                </form>
+            </div>
+        );
+    }
+
     if (commentsLoading) {
         return <h3>Loading...</h3>;
```

Comment:
you have this loading message, but when viewing an article with no comments in slow 3G the message I got instead was "This review has no comments..." , it might be that we're never getting into this if statement. It would be great if you do provide this message when comments are just loading still